README.md

```diff
@@ -1,162 +1,114 @@
-  - Made the Unity Environment available (see link on main page).
-  - Many improvements to documentation and examples.
-  - Upgraded to Mlagents 2.1.0-exp.1 (ml-agents python version 0.27.0).
-  - Fixed various bugs.
-- v2.2.3
-  - Now you can specify multiple different arenas in a single yml config file ant the environment will cycle through them each time it resets.
-- v2.2.2 
-  - Low quality version with improved fps. (will work on further improvments to graphics & fps later).
-- v2.2.1
-  - Improve UI scaling wrt. screen size.
-  - Fixed an issue with cardbox objects spawning at the wrong sizes.
-  - Fixed an issue where the environment would time out after the time period even when health > 0 (no longer intended behaviour).
-  - Improved Death Zone shader for weird Zone sizes.
-- v2.2.0 Health and Basic Scripts.
-  - Switched to health-based system (rewards remain the same).
-  - Updated overlay in play mode.
-  - Allow 3D hot zones and death zones and make them 3D by default in old configs.
-  - Added rewards that grow/decay (currently not configurable but will be added in next update).
-  - Added basic Gym Wrapper.
-  - Added basic heuristic agent for benchmarking and testing.
-  - Improved all other python scripts.
-  - Fixed a reset environment bug when resetting during training.
-  - Added the ability to set the DecisionPeriod (frameskip) when instantiating and environment.
-- v2.1.1 bugfix
-  - Fixed raycast length being less then diagonal length of standard arena.
-- v2.1 beta release
-  - Upgraded to ML-Agents release 2 (0.26.0).
-  - New features:
-    - Added raycast observations
-    - Added agent global position to observations
-
-### Notice
-
-   Copyright 2023 Kinds of Intelligence, Centre for the Future of Intelligence, University of Cambridge
-   
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
+Juliani, A., Berges, V., Vckay, E., Gao, Y., Henry, H., Mattar, M., Lange, D. (2018). [Unity: A General Platform for Intelligent Agents.](https://arxiv.org/abs/1809.02627) *arXiv preprint arXiv:1809.02627*
+
+Documentation for [ML-Agents](https://github.com/Unity-Technologies/ml-agents) should be consulted if you want additional resources.
+
+
+## The Animal-AI Community 
+
+Animal-AI has been an open-source research project from the beginning, and will continue to be so in the future. We welcome contributions from the community from all backgrounds and experiences, and we are always looking for new ways to collaborate. 
```

Comment:
nit: Would link to the contribution docs here

Reply:
done